README.md

```diff
@@ -0,0 +1,76 @@
+<p align="center">
+  <span">
+    <img src="https://raw.githubusercontent.com/parseablehq/.github/main/images/logo.svg#gh-light-mode-only" alt="Parseable" width="500" height="100" />
+    <img src="https://raw.githubusercontent.com/parseablehq/.github/main/images/logo-dark.png#gh-dark-mode-only" alt="Parseable" width="500" height="100" />
+  </a> 
+</p>
+
+<p align="center">
+  <a href="https://fossunited.org/" target="_blank"><img src="http://fossunited.org/files/fossunited-badge.svg"></a>
+  <img src="https://img.shields.io/github/commit-activity/m/parseablehq/parseable" alt="commits activity monthly">
+  <a href="https://launchpass.com/parseable" target="_blank"><img src="https://img.shields.io/badge/join%20slack-parseable-brightgreen.svg" alt="join slack"></a>
+  <a href="https://github.com/parseablehq/parseable/stargazers" target="_blank"><img src="https://img.shields.io/github/stars/parseablehq/parseable?style=social" alt="Github stars"></a>
+  <a href="https://twitter.com/parseableio" target="_blank"><img src="https://img.shields.io/twitter/follow/parseableio" alt="Twitter"></a>
+</p>
+
+<h4 align="center">
+  <a href="https://www.parseable.io/docs/quick-start" target="_blank">Quick Start</a> |
+  <a href="https://www.parseable.io/docs/introduction" target="_blank">Documentation</a> |
+  <a href="https://demo.parseable.io" target="_blank">Live Demo</a>
+  <br>
+</h4>
+
+Parseable is a cloud native, log storage and analysis platform. Parseable is indexing free by design. Written in Rust, Parseable can be deployed on Baremetal, VMs and Kubernetes.
+
+It ingests log data via HTTP POST calls and exposes a query API to search and analyze logs. It is compatible with logging agents like FluentBit, LogStash, FileBeat among others.
+
+<img src="https://raw.githubusercontent.com/parseablehq/.github/main/images/parseable.png" />
+
+
+
+## Getting Started
+
+1. Clone the repository.
+2. Run `npm install` to install all the dependencies.
+3. Run `npm run client` to start the console.
+4. Open `http://localhost:3000` in your browser.
+5. You can see the console running in your browser now :)
+6. For dashboard Access Enter the below credentials.
```

Comment:
This section is incomplete right now, because even if the UI is running on `localhost:3000` they can't really use it. 

We need to talk about getting started by setting the endpoint to demo.parseable somewhere.